src/theme/DocCard/styles.module.css

```
@@ -0,0 +1,32 @@
+.cardContainer {
```

```suggestion
/* 
  Why is this swizzled?
    See the associated DocCard/index.js for details. There are no customizations in this file.
*/

.cardContainer {
```

Blocking: 

I _think_ this message is accurate, and that you didn't make any changes in this file. Regardless, we should also specify what's custom in this file to make updates easier.

---

Thank you for noting this!